pkg/scheduler/scheduler.go

```diff
@@ -146,6 +148,19 @@ func WithEnableSchedulerEstimator(enableSchedulerEstimator bool) Option {
 	}
 }
 
+// WithGRPC sets the grpc config for scheduler
+func WithGRPC(port int, certFile, keyFile, trustedCAFile string, insecureSkipVerify bool) Option {
```

Comment:
```suggestion
func WithSchedulerEstimatorConnection(port int, certFile, keyFile, trustedCAFile string, insecureSkipVerify bool) Option {
```
Would it be more straightforward?